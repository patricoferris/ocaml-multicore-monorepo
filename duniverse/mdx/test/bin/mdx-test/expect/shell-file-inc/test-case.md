Mdx can include text from any file:


```sh file=script.sh
```
