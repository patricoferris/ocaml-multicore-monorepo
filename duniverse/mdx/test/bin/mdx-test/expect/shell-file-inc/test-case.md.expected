Mdx can include text from any file:


```sh file=script.sh
$ sudo rm -rf /
```
